app/models/local_authority.rb

```diff
@@ -4,9 +4,25 @@ class LocalAuthority < ApplicationRecord
   has_many :users, dependent: :destroy
   has_many :planning_applications, dependent: :destroy
 
-  validates :council_code, :name, :subdomain, presence: true
+  validates :subdomain, :signatory_name, :signatory_job_title, :enquiries_paragraph, :email_address, :feedback_email,
+            presence: true
+
+  enum subdomain: {
+    lambeth: "lambeth",
+    southwark: "southwark",
+    buckinghamshire: "buckinghamshire",
+    ripa: "ripa"
+  }
 
   def signatory
     "#{signatory_name}, #{signatory_job_title}"
   end
+
+  def council_code
+    I18n.t("council_code.#{subdomain}")
+  end
+
+  def council_name
+    ripa? ? subdomain.capitalize : "#{subdomain.capitalize} Council Authority"
```

Comment:
I also thought of this but since all of these would just have `Council Authority` for now at the end I think is ok. Maybe in the future when we find that some other councils have different name then we can change slighting a bit our locales files maybe to this:
```
councils:
  lambeth:
    name: Lambeth Council Authority
    code: LBH
  southwark:
    name: Southwark Council Authority
    code: SWK
  buckinghamshire:
    name: Buckinghamshire Council Authority
    code: BUC
  ripa:
    name: Ripa
    code: RIPA
```